src/main/java/com/enterprise/redcord/dao/MessageDAOStub.java

```diff
@@ -25,7 +25,11 @@ public Message saveMessage(Message messageEntry) {
             latestMessageId = lastMessage.getMessageId();
         }
         messageEntry.setMessageId(latestMessageId + 1);
-        allMessages.put(messageEntry.getMessageId(), messageEntry);
+        try {
+            allMessages.put(messageEntry.getMessageId(), messageEntry);
+        } catch(Exception e) {
+            e.printStackTrace();
```

Comment:
allMessages is simply a HashMap.  The put () method won't throw an exception.  So, this is not needed.